### PR TITLE
Draft for release T7.3

### PR DESCRIPTION
**Pull request by aesedepece**
**WARNING**: this is to be merged STRICTLY after #1126 and #1136.

### CODE REVIEW

File: witnet.toml [hunk 1]
```diff
@@ -2,7 +2,35 @@
...
+    "167.86.116.89:21337",
+    "173.212.212.43:21337",
+    "164.68.100.118:21337",
+    "144.217.204.0:21337",
+    "167.86.76.5:21337",
+    "217.254.186.98:21337",
+    "73.153.225.122:21337",
+    "41.212.58.87:21337",
+    "144.202.100.245:21337"
+]
```
Comment (aesedepece) [inline witnet.toml:33]: Node operators are going to love this.

**Comment by lrubiorod** [comment]
1Million to twit1jqgf4rxjrgas3kdhj3t4cr3mg3n33m8zw0aulr pease :P

**Comment by lrubiorod** [comment]
Should we add the public_addr field in witnet.toml to encourage their use?

**Comment by aesedepece** [comment]
> Should we add the public_addr field in witnet.toml to encourage their use?

This is supposed to be rebased on top of your PR, and then the field will magically appear ;)

**Comment by aesedepece** [comment]
Ok, it looks like I wrongly assumed PR #1136 to modify the `witnet.toml` file, but it did not. I will push an additional commit with the change.

File: witnet.toml [hunk 2]
```diff
@@ -2,7 +2,36 @@
 # https://docs.witnet.io/configuration/toml-file/
 [connections]
 server_addr = "0.0.0.0:21337"
-known_peers = ["52.166.178.145:21337", "52.166.178.145:22337", "52.166.178.145:23337", "52.166.178.145:24337", "52.166.178.145:25337"]
+public_addr = "0.0.0.0:21337"
```
Comment (tmpolaczyk) [inline witnet.toml:5]: A comment here would be helpful, to explain the difference between `server_addr` and `public_addr`.

**Pull request merged**